index.bs

```diff
@@ -2034,7 +2035,7 @@ ignore>command parameters</var> are:
 
    <dl>
       <dt>"ready"</dt>
-      <dd>The [=remote end=]’s readiness state.</dd>
+      <dd>The [=remote end=]’s [=readiness state=].</dd>
 
       <dt>"message"</dt>
       <dd>An implementation-defined string explaining the [=remote end=]’s readiness
```

Comment:
Can you link this one too?

Reply:
Oh, that one I completely missed because of the line brake. Thanks for pointing out. I will have it fixed in my next push. I wonder if the name might cause confusion in regards of the newly added `ReadinessState` for navigation?